PACKAGE = "seg_rslidar_param"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pass_limit", double_t, 0, "", -1.0, -4.0, 0.0);
gen.add("planar_threshold", double_t, 0, "", 0.3, 0.01, 0.5)
gen.add("cluster_tolerance", double_t, 0, "", 0.8, 0.0, 100.0)

exit(gen.generate(PACKAGE, "seg_rslidar_param", "SegRslidar"))